config/src/config/upstream_config.rs

```diff
@@ -1,43 +1,42 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::network_id::{NetworkId, NodeNetworkId};
+use crate::network_id::NetworkId;
```

Comment:
I wonder if `config` is the right place for this? Maybe we should just stuff these in `diem-types` lol

Reply:
Probably not, but I think we can rethink the location afterwards, especially depending on what we use the `PeerNetworkId` for.

The `UpstreamConfig` is gone, but the remnants are left